components/MapsDedi/index.vue

```diff
@@ -41,15 +41,97 @@
           </div>
         </div>
       </div>
+      <div v-show="isDesa" class="maps">
+        <div class="maps__motif">
+          <img src="~/assets/images/MotifDot.svg" alt="Motif Dot">
+        </div>
+        <div
+          ref="boxmaps"
+          :class="{
+            'maps__boxmaps--fullscreen': isFullscreen,
+            'maps__boxmaps--exitfullscreen': !isFullscreen,
+            'maps__boxmaps': true,
+
+          }"
+        >
+          <div
+            ref="maps"
+            :class="{
+              'maps__boxmaps-map--fullscreen': isFullscreen,
+              'maps__boxmaps-map--exitfullscreen': !isFullscreen,
+              'maps__boxmaps-map': true
+
+            }"
+          />
+          <div
+            :class="{
+              'maps__boxmaps-info':true,
+              'maps__boxmaps-info--fullscreen': isFullscreen,
+              'maps__boxmaps-info--exitfullscreen': !isFullscreen,
+            }"
+          >
+            <div class="maps__boxmaps-info-box">
+              <div class="maps__boxmaps-info-box-title">
+                Total <span class="font-bold">{{ totalVillage }} Desa</span> telah bergabung
```

Comment:
super minor:

I think instead of using `<span class="font-bold">` we can use `<strong>` because it has more semantic meaning 🤔

Reply:
alright i will refactor it